drud/ddev
# Allow home directory additions in web container, fixes #926, fixes #1459

## The Problem/Issue/Bug:

#926 requests a way to configure the user inside the web container (.bashrc, etc)
#1459 specifically requests a way to set the user's ~/.gitconfig as they apparently do commits inside the container

## How this PR Solves The Problem:

Copy the contents of the project's .ddev/homeadditions into the containers ~/

## Manual Testing Instructions:
```
mkdir -p  .ddev/homeadditions
touch .ddev/homeadditions/ITWORKED.txt
ddev start
ddev ssh
ls -al ~/
```
You should see your ITWORKED.txt

Try out some of the example usages in [docs](https://github.com/rfay/ddev/blob/6eb34b328d4e4ef36419e61b0161313f91ba29dc/docs/users/extend/in-container-configuration.md)

Review docs. 

## Automated Testing Overview:

Hopefully existing tests would catch regressions.

## Related Issue Link(s):

## Release/Deployment notes:
